GherkinSyncTool/README.md

```diff
@@ -0,0 +1,90 @@
+# GherkinSyncTool
+
+GherkinSyncTool is an open-source console application, that synchronizes tests scenarios in
+a [gherkin syntax](https://cucumber.io/docs/gherkin/) also known as feature files with a tests management system or any
+other destinations. Currently, a TestRail synchronizer is implemented
```

Comment:
I would introduce a section with list of supported test management systems instead of phrase 
"Currently, a TestRail synchroniser is implemented"

Reply:
Fixed